### PR TITLE
Morphology segment combine update

### LINKED ISSUE #928: segment_combine() fails with ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
segment_combine() fails with `ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()`

code to recreate:
```
import numpy as np
from plantcv import plantcv as pcv

#works:
segs = [np.array([[[1,2]], [[3,4]]]), np.array([[[5,6]], [[7,8]]]), np.array([[[9,10]], [[11,12]]])]
pcv.morphology.segment_combine([0,1], segs, np.zeros((100,100),dtype=np.uint8))

#fails:
segs = [np.array([[[1,2]], [[3,4]]]), np.array([[[5,6]], [[7,8]]]), np.array([[[9,10]], [[11,12]]])]
pcv.morphology.segment_combine([1,2], segs, np.zeros((100,100),dtype=np.uint8))
```

full error:
```
---------------------------------------------------------------------------

ValueError                                Traceback (most recent call last)

[<ipython-input-101-7ac3bd532796>](https://localhost:8080/#) in <module>()
      8 #fails:
      9 segs = [np.array([[[1,2]], [[3,4]]]), np.array([[[5,6]], [[7,8]]]), np.array([[[9,10]], [[11,12]]])]
---> 10 pcv.morphology.segment_combine([1,2], segs, np.zeros((100,100),dtype=np.uint8))

[/usr/local/lib/python3.7/dist-packages/plantcv/plantcv/morphology/segment_combine.py](https://localhost:8080/#) in segment_combine(segment_list, objects, mask)
     42         new_objects = all_objects[segment_list[0]]
     43         # Remove the objects getting combined from the list of all objects
---> 44         all_objects.remove(objects[segment_list[0]])
     45 
     46         while count < num_contours:

ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```

Local environment:
 - OS: Linux
 - Environment: google colab, python 3.7
 - PlantCV Version 3.14.1

### PR DESCRIPTION
**Pull request by HaleySchuhl**
**Describe your changes**
Update the functionality to the `pcv.morphology.segment_combine`. This repairs the error with using `.remove` in issue #928 and simplifies the code so that it will only accept one list of segmentss to get combined. 

**Type of update**
Is this a:
* Bug fi

**Associated issues**
- closes #928 

**Additional context**

**For the reviewer**
See [this page](https://plantcv.readthedocs.io/en/stable/pr_review_process/) for instructions on how to review the pull request.
- [x] PR functionality reviewed in a Jupyter Notebook
- [x] All tests pass
- [x] Test coverage remains 100%
- [x] Documentation tested
- [x] New documentation pages added to `plantcv/mkdocs.yml`
- [x] Changes to function input/output signatures added to `updating.md`
- [x] Code reviewed
- [x] PR approved

### CODE REVIEW

**Comment by eyaler** [comment]
Thanks! small comment:
```
if type(segment_list[0]) is not int:
```
does not gracefully handle the case of an empty segment_list

**Comment by HaleySchuhl** [comment]
> Thanks! small comment:
> 
> ```
> if type(segment_list[0]) is not int:
> ```
> 
> does not gracefully handle the case of an empty segment_list

Very true! An added `fatal_error` with an informative message is a good idea.

File: docs/segment_combine.md [hunk 1]
```diff
@@ -7,14 +7,14 @@ Manually combine segments from a segmented skeleton
 **returns** Labeled image, new segment objects list 
 
 - **Parameters:**
-    - segment_list - List of segments to get combined, or list of lists of segments to get combined
+    - segment_list - List of segments to get combined to get combined
```
Comment (nfahlgren) [inline docs/segment_combine.md:10]:
```suggestion
    - segment_list - List of segments to get combined
```

**Review by nfahlgren — CHANGES_REQUESTED** [review]

**Review by nfahlgren — APPROVED** [review]
Thanks!

**Pull request merged**